docs/for-ops/upgrade-notes.md

```diff
@@ -121,14 +128,22 @@ kubectl scale sts harbor-redis --replicas=1 -n harbor
 
 #### Wait for pod ready status
 :::note
-If the version you are going to upgrade to is of v4.7.0 or higher you also have to scale up the apl-operator
+If the version you are going to upgrade to is of v4.7.0 or higher you also have to enable automated syncing and scale up the apl-operator
 :::
 
 ```shell
 kubectl scale deploy apl-operator --replicas=1 -n apl-operator
 ```
 
+```shell
+kubectl patch application apl-operator-apl-operator \
```

Comment:
It seems that `allowEmpty": false` is missing. 

 "automated": {
    "allowEmpty": false,
    "prune": true,
    "selfHeal": true
  },